### PR TITLE
task6 commit

### PR DESCRIPTION
**Pull request by impenetrabl1**
PR feature6 - develop

### CODE REVIEW

File: task6.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+def func6(n):
+    num = [i ** i for i in range(1, n + 1)]
+    str1 = str(sum(num))
+    return str1[-10:]
+
+
+start = time.time()
+print(func6(n))
+print((time.time() - start))
```
Comment (MaksimLion) [inline task6.py]: yesterday, i sad about decorators, take advantage of it

File: task6.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import time
+
+n = 10000
+
+
+def func6(n):
+    num = [i ** i for i in range(1, n + 1)]
+    str1 = str(sum(num))
```
Comment (MaksimLion) [inline task6.py]: use more readable names for variable

File: task6.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        value = func(arg)
+        print((time.time() - start))
+        return value
+    return wrapper
+
+
+@function_time
+def calc_range(n):
+    squares = [i ** i for i in range(1, n + 1)]
+    sum1 = str(sum(squares))
```
Comment (AlehSivakou) [inline task6.py]: please avoid short namings

calc_range should be calculate_range
sum1 is not an option critically. name it simply sum

File: task6.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        value = func(arg)
+        print((time.time() - start))
+        return value
+    return wrapper
+
+
+@function_time
+def calculate_range(n):
+    squares = [i ** i for i in range(1, n + 1)]
+    sum = str(sum(squares))
```
Comment (MaksimLion) [inline task6.py]: sum this is specific name, add come prefix

File: task6.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    return sum[-10:]
+
+
+with FileOpen(r'C:\Users\Aliaksandr.Baryhin\Desktop\dataFiles\task6_data.txt', 'a') as open_file:
+    open_file.write('\nLast 10 numbers of range: ' + str(calculate_range(number)))
```
Comment (MaksimLion) [inline task6.py]: you can use "{}".format()  instead string concatenation

**Pull request merged**